app/gui2/src/components/GraphEditor/GraphNode.vue

```diff
@@ -114,7 +114,7 @@ const warning = computed(() => {
   const info = projectStore.computedValueRegistry.db.get(externalId)
   const warning = info?.payload.type === 'Value' ? info.payload.warnings?.value : undefined
   if (!warning) return
-  return '⚠ Warning: ' + warning!
+  return '⚠\xa0\xa0Warning: ' + warning!
```

Comment:
Let's use a named constant for `\xa0`, some of us (like me) are HTML noobs who can't recognize an *nbsp* by its hex code.

Reply:
good catch, thanks!